scalability_and_performance/telco_ref_design_specs/core/telco-core-ref-design-components.adoc

```diff
@@ -75,6 +76,10 @@ include::modules/telco-core-power-management.adoc[leveloffset=+1]
 [role="_additional-resources"]
 .Additional resources
 
+* xref:../../../rest_api/node_apis/performanceprofile-performance-openshift-io-v2.adoc#spec-workloadhints[Performance Profile]
+
+* link:../../../scalability_and_performance/low_latency_tuning/cnf-tuning-low-latency-nodes-with-perf-profile.adoc#cnf-configuring-power-saving-for-nodes_cnf-low-latency-perf-profile[Configuring power saving for nodes]
```

Comment:
Page not found with this xref.

Reply:
This is a link, should be an xref. Updated.